extension/addreactionparticipants.js

```diff
@@ -0,0 +1,60 @@
+const addReactionParticipants = currentUser => {
+	$('.comment-reactions').each((index, reactionsContainer) => {
+		const $reactionsContainer = $(reactionsContainer);
+		if (!$reactionsContainer.hasClass('has-reactions')) {
+			return;
+		}
+
+		const $reactionButtons = $reactionsContainer.find('.comment-reactions-options .reaction-summary-item[aria-label]');
+
+		$reactionButtons.each((index, element) => {
+			element.addEventListener('click', () => {
```

Comment:
OK, this is added in. Now there's one event listener and it's only applied once. Before it was reapplying the event listeners every time you clicked.